common/src/main/generated/resources/assets/endermanoverhaul/lang/en_us.json

```diff
@@ -3,6 +3,7 @@
   "config.endermanoverhaul.allowPickingUpBlocks": "Allow Picking Up Blocks",
   "config.endermanoverhaul.allowSpawning": "Allow Spawning",
   "config.endermanoverhaul.endEndermanTeleportChance": "End Enderman Teleport Chance",
+  "config.endermanoverhaul.endEndermanTeleportCooldown": "End Enderman Teleport Cooldown",
```

Comment:
This will be replaced every time datagen is run. Must be put in the [ModLangProvider](https://github.com/bonsaistudi0s/Enderman-Overhaul/blob/1.20.x/neoforge/src/main/java/tech/alexnijjar/endermanoverhaul/datagen/provider/client/ModLangProvider.java). then run datagen